src/utils/parser.js

```diff
@@ -0,0 +1,2252 @@
+const jsdom = require("jsdom");
+const { JSDOM } = jsdom;
+
+const xmlData = `<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n
```

Comment:
We can use the graphql query in my gatsby-node.js file to parse through the entities.xml file